esti/lakectl_local_test.go

```diff
@@ -437,12 +437,15 @@ func TestLakectlLocal_commit(t *testing.T) {
 			presign := fmt.Sprintf(" --pre-sign=%v ", tt.presign)
 			RunCmdAndVerifyContainsText(t, Lakectl()+" local clone lakefs://"+repoName+"/"+vars["BRANCH"]+"/"+vars["PREFIX"]+presign+dataDir, false, "Successfully cloned lakefs://${REPO}/${REF}/${PREFIX} to ${LOCAL_DIR}.", vars)
 
-			//relPath, err := filepath.Rel(tmpDir, dataDir)
-			//require.NoError(t, err)
-
 			RunCmdAndVerifyContainsText(t, Lakectl()+" local status "+dataDir, false, "No diff found", vars)
 
 			// Modify local folder - add and remove files
+			os.MkdirAll(filepath.Join(dataDir, "subdir"), os.ModePerm)
+			os.MkdirAll(filepath.Join(dataDir, "subdir-a"), os.ModePerm)
+			fd, err = os.Create(filepath.Join(dataDir, "subdir", "test.txt"))
+			require.NoError(t, err)
+			fd, err = os.Create(filepath.Join(dataDir, "subdir-a", "test.txt"))
+			require.NoError(t, err)
```

Comment:
I think this is missing verification for this test case (on the status)

Reply:
At the end of the test, we verify that lakectl status returned no diff.
In the original bug, we've got changes after committing

Reply:
Understood, but we still need to fix the verify status output